src/main/java/com/romraider/io/connection/SerialConnectionProperties.java

```diff
@@ -42,10 +42,11 @@ public int getBaudRate() {
         return baudRate;
     }
 
-    public void setBaudRate(int b) {
-
-    }
-
+	public void setBaudRate(int b) {
```

Comment:
I don't see any references to this setter, can you restore this class?  This class is meant to be constructed with the appropriate values on creation.